tty/imx6ull-uart/imx6ull-uart.c

```diff
@@ -704,6 +704,9 @@ int main(int argc, char **argv)
 		create_dev(&dev, _PATH_CONSOLE);
 
 	libklog_init(libklog_clbk);
+#else
+	(void)is_console;
+	(void)libklog_clbk;
```

Comment:
IMHO the whole function definition should be under ifdef. This approach might make compiler to keep the function in the final `elf`

Reply:
Ok, you are right. I wanted to avoid multiple ifdefs in the file, but I can do this the way you suggest.